libs/platform/src/lib/components/footer/footer.component.scss

```diff
@@ -86,11 +46,11 @@ $fd-footer-font-size: var(--sapFontSize, 0.875rem);
         }
         &--link {
             color: $fd-link-font-color;
-            font-family: '72', '72full', Arial, Helvetica, sans-serif;
+            font-family: var(--sapFontFamily, '72', '72full', Arial, Helvetica, sans-serif);
             font-size: 0.875rem;
-            display: inherit;
-            padding-top: 0.75rem;
-            width: 73%;
+            display: inline;
+
+            width: 75%;
             > div > * {
```

Comment:
[this comment is unresolved](https://github.com/SAP/fundamental-ngx/pull/4936#discussion_r597383325), can you address it?

Reply:
this is necessary, since we are injecting template we are unaware about the elements being injected so * is needed for adding margin to the unknown child elements.